grpc-async.opam

```diff
@@ -8,11 +8,12 @@ authors: [
   "Andrew Jeffery <dev@jeffas.io>, Daniel Quernheim <quernd@users.noreply.github.com>, Michael Bacarella <m@bacarella.com>, Sven Anderson <sven@anderson.de>, Wojtek Czekalski <me@wczekalski.com>, dimitris.mostrous <dimitris.mostrous@gmail.com>"
```

Comment:
Please add yourself to the list of authors in all the .opam files

Reply:
Done d2c7a251ae4c5885fc46bf3429728ced3b5151cc